### DISCUSSION TITLE
Slash commands again...

### DISCUSSION THREAD

**Discussion by Tumpa-Prizrak** [Rapptz/discord.py#8050] [Q&A]
In the nextcord was this commands
```py
await bot.register_new_application_commands()
```
It register slash commands witch don't was before in bot, but was in code.
After it new slash commands added almost immediately.
I just add new commands and it isn't added after ten minutes. Is in the d.py has something like that?

**Comment by imvaskel** [comment]
You want to ``sync`` your ``CommandTree``. It is likely nextcord was registering guild commands, which is instant, compared to global's 1h sync time. If you are looking to sync a certain guild, then you will want to pass the ``guild`` kwarg (which needs to be a ``Snowflake``).

For Reference: https://discordpy.readthedocs.io/en/latest/interactions/api.html#discord.app_commands.CommandTree.sync

**Reply by ooliver1** [reply]
Discord's new backend v2 registers globals almost immediately, the old 1h time was not a static limit and seemed to have been disabled at non-peak times

**Reply by imvaskel** [reply]
> Discord's new backend v2 registers globals almost immediately, the old 1h time was not a static limit and seemed to have been disabled at non-peak times

To my knowledge, the new backend V2 is still in the rollout phase and not everyone has access to it.

**Reply by dolfies** [reply]
Everyone has access to it. Only pre-RNA Android clients are not currently utilizing it.

**Comment by Rapptz** [comment]
The way to sync commands is [`await bot.tree.sync()`](https://discordpy.readthedocs.io/en/latest/interactions/api.html#discord.app_commands.CommandTree.sync). This registers globals commands. For guild commands you do `await bot.tree.sync(guild=obj)` where `obj` is a type that has an `.id` attribute such as `Guild` or `Object`.
